contents/handbook/people/team-structure/design.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Team Design
+sidebar: Handbook
+showTitle: true
+hideAnchor: true
+---
+
+## People
+
+- [Cory Watilo (Team lead, Lead Designer)](/handbook/company/team/#cory-watilo-lead-designer)
+- [Lottie Coxon, Graphic Design)](/handbook/company/team#lottie-coxon-graphic-designer)
+- Mike Nicklas, Front End Engineer
+
+## Mission
+
```

Comment:
@corywatilo what would you see as the mission of design? I've put responsibilities etc below as we've talked about before and largely as you had defined them previously